_posts/2022-02-15-label-noise-intro.md

```diff
@@ -0,0 +1,121 @@
+# Introduction
+
+This blog will introduce the label noise problem in our datasets and cover various cleaning methods that can reduce the time taken to fix noisy labels in a dataset.
```

Comment:
Can we do better intro? With more hook? While this is adequate, I don't find it interesting enough to read. And I am not asking to go the full medium.com style, but to describe the problem and motivation first, and why it's important. Right now it looks like a Skit internal journal entry.